eq-author-api/src/validation/customKeywords/validateSurveyId.js

```diff
@@ -35,8 +37,40 @@ module.exports = (ajv) =>
           ),
         ];
         return false;
-      }
+      } else {
+        if (
+          typeof questionnaire.surveyId === "string" &&
+          questionnaire.surveyId.length > 0 &&
+          !questionnaire.surveyId.match(/^\d{3}$/)
+        ) {
+          isValid.errors = [
+            createValidationError(
+              instancePath,
+              fieldName,
+              ERR_INVALID,
+              questionnaire,
+              ERR_INVALID
+            ),
+          ];
+          return false;
+        } else if (
```

Comment:
@Farhanam76 The 'else if' block from lines 56 to 71 must instead be the first 'if'  block so that it will check the empty error condition first.